universal-application-tool-0.0.1/app/views/applicant/ApplicantProgramBlockEditView.java

```diff
@@ -150,22 +156,66 @@ private Tag renderFileUploadBlockSubmitForm(Params params) {
             .setSignedFileUploadRequest(signedRequest)
             .build();
 
-    return form()
-        .withId("cf-block-form")
-        .attr(ENCTYPE, "multipart/form-data")
-        .withAction(signedRequest.actionLink())
-        .withMethod(HttpVerbs.POST)
-        .with(
-            each(
-                params.block().getQuestions(),
-                question -> renderQuestion(question, rendererParams)))
-        .with(renderFileUploadBottomNavButtons(params));
+    Tag uploadForm =
+        form()
+            .withId(BLOCK_FORM_ID)
+            .attr(ENCTYPE, "multipart/form-data")
+            .withAction(signedRequest.actionLink())
+            .withMethod(HttpVerbs.POST)
+            .with(
+                each(
+                    params.block().getQuestions(),
+                    question -> renderQuestion(question, rendererParams)));
+    Tag skipForms = renderSkipFileUploadForms(params);
+    Tag buttons = renderFileUploadBottomNavButtons(params);
+    return div(uploadForm, skipForms, buttons);
+  }
+
+  private Tag renderSkipFileUploadForms(Params params) {
```

Comment:
(blocking) this renders skip and delete forms. please update name, and maybe add a comment if you think that's helpful.

Reply:
Done. PTAL